src/Sajari/Client/Client.php

```diff
@@ -0,0 +1,395 @@
+<?php
```

Comment:
@tbillington I think this namespace should just be `Sajari` i.e. file exists at `src/Sajari/Client.php` - no need to stutter the `Client`, yeah?